File: CoreJava/collage-app/src/com/techlab/person/Person.java
package com.techlab.person;

import java.text.*;
import java.util.Date;

public class Person {

	protected int id;
	protected String name;
	protected String address;
	protected Date dateOfBirth;

//	constructor
	public Person(int id, String name, String address, Date dateOfBirth) {
		this.id = id;
		this.name = name;
		this.address = address;
		this.dateOfBirth = dateOfBirth;
	}

//	getters
	public int getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	public String getAddress() {
		return address;
	}

	public String getDateOfBirth() {
		DateFormat df = new SimpleDateFormat("dd/MM/yyyy");
		return df.format(dateOfBirth);
	}

}

File: C#Folder/CollegeApp/CollegeApp/Model/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CollegeApp.Model
{
    class Person
    {
        private int _id;
        private string _address;
        private DateTime _dateOfBirth;

        public Person(int id, string address, DateTime dateOfBirth)
        {
            this._id = id;
            this._address = address;
            this._dateOfBirth = dateOfBirth;
        }

        public int Id { get => _id; }

        public string Address { get => _address; }

        public DateTime DateOfBirth { get => _dateOfBirth; }
    }
}
